doc/cvodes/guide/source/Usage/SIM.rst

```diff
@@ -1307,7 +1311,22 @@ using the current :math:`\gamma` value.
      * ``CVLS_ILL_INPUT`` -- The frequency ``msbj`` is negative.
 
    **Notes:**
-      The Jacobian update frequency is only checked within calls to the  linear solver setup routine, as such values of ``msbj`` < ``msbp`` will  result in recomputing the Jacobian every ``msbp`` steps. See :c:func:`CVodeSetLSetupFrequency` for setting the linear solver setup frequency  ``msbp``.  If ``msbj = 0``, the default value of 51 will be used. Otherwise an error is returned. This function must be called after the CVLS linear solver interface has been initialized through a call to :c:func:`CVodeSetLinearSolver`.
+      If ``nstlj`` is the step number at which the Jacobian information was
+      lasted updated and ``nst`` is the current step number,
+      ``nst - nstlj >= msbj`` indicates the Jacobian information will be updated
+      during the next linear solver setup call.
+
+      As the Jacobian update frequency is only checked *within* calls to the
+      linear solver setup routine, Jacobian information may be more than
+      ``msbj`` steps old when updated depending on when a linear solver setup
+      call occurs. See :numref:`CVODES.Mathematics.nls` for more information on
+      when linear solver setups are preformed.
+
+      If ``msbj = 0``, the default value of 51 will be used. Otherwise an error
```

Comment:
Is this really 51?

Reply:
Yes, see [this comment](https://github.com/LLNL/sundials/pull/250/files#r1074006308).